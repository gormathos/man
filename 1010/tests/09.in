sol_id;user_id;task_id;lang_id;score
170493;1087;2005;2;100.00
170494;1034;2004;1;100.00
170496;1033;2004;4;10.42
170499;1044;2002;5;0.00
170503;1012;2001;2;38.64
170508;1027;2003;1;0.00
170514;1042;2002;4;42.31
170521;1037;2005;5;0.00
170529;1044;2003;5;71.11
170538;1031;2001;2;100.00
170548;1082;2003;2;0.00
170559;1058;2005;3;100.00
170571;1080;2004;5;0.00
170584;1025;2003;1;100.00
170598;1097;2003;2;100.00
170613;1091;2002;3;0.00
170629;1096;2002;2;100.00
170646;1099;2002;5;0.00
170664;1009;2002;4;23.08
170683;1001;2004;5;85.42
170703;1007;2003;4;0.00
170724;1068;2004;5;0.00
170746;1085;2002;3;100.00
170769;1077;2005;4;70.83
170793;1075;2003;4;13.33
170818;1060;2005;4;100.00
170844;1080;2002;4;3.85
170871;1028;2004;1;20.83
170899;1028;2005;1;100.00
170928;1034;2005;3;12.50
170958;1002;2004;1;25.00
170989;1007;2004;2;0.00
171021;1093;2003;2;73.33
171054;1056;2002;1;100.00
171088;1017;2001;1;0.00
171123;1087;2001;1;0.00
171159;1097;2003;5;0.00
171196;1046;2002;5;34.62
171234;1064;2001;5;54.55
171273;1013;2004;1;54.17
171313;1034;2003;2;60.00
171354;1022;2003;5;0.00
171396;1011;2004;1;45.83
171439;1022;2001;4;86.36
171483;1017;2001;4;88.64
171528;1012;2004;2;100.00
171574;1059;2001;4;72.73
171621;1043;2002;1;46.15
171669;1055;2001;4;88.64
171718;1034;2003;1;100.00
171768;1049;2003;1;0.00
171819;1041;2001;1;47.73
171871;1019;2004;2;6.25
171924;1039;2005;3;58.33
171978;1092;2002;3;100.00
172033;1033;2003;3;33.33
172089;1026;2002;2;26.92
172146;1012;2002;4;0.00
172204;1021;2003;1;44.44
172263;1033;2005;3;16.67
172323;1037;2002;2;0.00
172384;1053;2005;1;25.00
172446;1081;2002;1;100.00
172509;1063;2001;2;36.36
172573;1072;2002;5;42.31
172638;1064;2005;2;8.33
172704;1026;2004;1;81.25
172771;1012;2004;5;100.00
172839;1078;2004;4;20.83
172908;1041;2004;3;100.00
172978;1088;2003;5;28.89
173049;1049;2002;4;53.85
173121;1030;2002;2;100.00
173194;1074;2005;3;83.33
173268;1032;2002;2;84.62
173343;1071;2004;1;6.25
173419;1006;2005;3;0.00
173496;1039;2003;5;8.89
173574;1043;2002;2;23.08
173653;1096;2005;3;0.00
173733;1042;2002;4;61.54
173814;1010;2001;5;20.45
173896;1083;2004;3;8.33
173979;1047;2001;5;15.91
174063;1089;2005;3;100.00
174148;1084;2004;5;100.00
174234;1080;2005;3;54.17
174321;1018;2003;5;62.22
174409;1038;2005;1;100.00
174498;1036;2005;1;100.00
174588;1035;2005;4;20.83
174679;1002;2005;4;100.00
174771;1027;2005;5;100.00
174864;1022;2002;1;0.00
174958;1009;2004;4;100.00
175053;1077;2002;2;100.00
175149;1085;2001;5;93.18
175246;1072;2001;5;100.00
175344;1037;2004;5;0.00
175443;1097;2005;4;100.00
175543;1005;2001;4;90.91
175644;1002;2001;4;0.00
175746;1096;2004;5;0.00
175849;1027;2001;1;0.00
175953;1062;2002;2;92.31
176058;1074;2005;3;0.00
176164;1059;2004;2;0.00
176271;1033;2005;1;100.00
176379;1029;2001;3;100.00
176488;1040;2002;5;0.00
176598;1068;2002;4;53.85
176709;1034;2004;5;0.00
176821;1027;2005;4;45.83
176934;1011;2004;3;41.67
177048;1030;2005;4;100.00
177163;1085;2001;1;84.09
177279;1012;2001;3;100.00
177396;1013;2002;5;88.46
177514;1061;2005;1;100.00
177633;1092;2003;2;0.00
177753;1031;2004;4;45.83
177874;1029;2001;4;20.45
177996;1050;2004;2;0.00
178119;1068;2001;2;0.00
178243;1071;2002;4;42.31
178368;1079;2005;5;54.17
178494;1091;2005;3;100.00
178621;1043;2004;5;100.00
178749;1014;2003;2;53.33
178878;1050;2004;1;100.00
179008;1004;2004;1;0.00
179139;1088;2004;1;83.33
179271;1028;2004;3;100.00
179404;1080;2005;3;100.00
179538;1053;2001;4;81.82
179673;1014;2001;1;6.82
179809;1072;2003;3;2.22
179946;1004;2004;4;97.92
180084;1082;2003;3;86.67
180223;1008;2002;2;100.00
180363;1048;2002;3;0.00
180504;1010;2001;5;70.45
180646;1058;2001;1;65.91
180789;1077;2003;1;0.00
180933;1049;2003;4;100.00
181078;1085;2004;2;47.92
181224;1038;2005;2;100.00
181371;1064;2001;3;0.00
181519;1072;2001;2;100.00
181668;1071;2003;2;95.56
181818;1095;2005;5;0.00
181969;1004;2002;3;100.00
182121;1001;2004;2;81.25
182274;1079;2005;5;0.00
182428;1083;2003;5;11.11
182583;1012;2002;3;0.00
182739;1039;2001;3;52.27
182896;1022;2005;2;100.00
183054;1061;2002;2;11.54
183213;1039;2005;1;87.50
183373;1004;2001;4;100.00
183534;1028;2003;3;100.00
183696;1052;2005;1;0.00
183859;1032;2002;2;0.00
184023;1084;2005;2;41.67
184188;1098;2004;1;22.92
184354;1088;2001;5;0.00
184521;1074;2005;1;4.17
184689;1018;2003;2;17.78
184858;1003;2005;3;100.00
185028;1063;2002;5;100.00
185199;1036;2002;2;100.00
185371;1001;2005;2;0.00
185544;1017;2002;2;73.08
185718;1044;2003;5;100.00
185893;1097;2004;3;6.25
186069;1048;2004;2;0.00
186246;1068;2001;1;0.00
186424;1059;2004;2;29.17
186603;1016;2005;2;29.17
186783;1057;2005;1;75.00
186964;1094;2003;3;100.00
187146;1014;2001;5;0.00
187329;1044;2005;3;95.83
187513;1016;2003;2;0.00
187698;1059;2003;3;0.00
187884;1044;2004;4;100.00
188071;1084;2005;2;100.00
188259;1020;2001;4;13.64
188448;1061;2002;2;65.38
188638;1036;2005;4;100.00
188829;1043;2002;5;0.00
189021;1031;2005;2;0.00
189214;1068;2002;2;69.23
189408;1090;2001;5;13.64
189603;1068;2001;2;11.36
189799;1010;2001;5;100.00
189996;1074;2003;3;57.78
190194;1083;2001;3;0.00
190393;1054;2002;5;88.46
190593;1090;2005;2;0.00
190794;1063;2002;1;100.00
190996;1013;2004;2;0.00
191199;1089;2003;3;68.89
191403;1035;2003;1;0.00
191608;1057;2004;3;0.00
191814;1048;2004;3;0.00
192021;1030;2005;2;0.00
192229;1075;2005;1;41.67
192438;1060;2004;1;100.00
192648;1073;2005;1;0.00
192859;1097;2003;4;53.33
193071;1083;2004;1;33.33
193284;1066;2001;5;0.00
193498;1085;2001;3;100.00
193713;1049;2002;3;42.31
193929;1072;2002;5;100.00
194146;1035;2004;4;0.00
194364;1058;2001;3;68.18
194583;1081;2002;2;65.38
194803;1022;2003;1;55.56
195024;1037;2004;1;0.00
195246;1056;2005;3;100.00
195469;1093;2002;4;38.46
195693;1084;2004;5;0.00
195918;1096;2005;2;95.83
196144;1040;2005;1;100.00
196371;1006;2001;5;0.00
196599;1004;2003;4;0.00
196828;1003;2001;5;100.00
197058;1024;2003;4;100.00
197289;1040;2002;5;11.54
197521;1046;2002;5;65.38
197754;1036;2005;3;100.00
197988;1091;2003;4;100.00
198223;1050;2001;2;100.00
198459;1013;2004;4;100.00
198696;1098;2004;3;27.08
198934;1085;2003;3;0.00
199173;1056;2004;1;0.00
199413;1100;2002;4;0.00
199654;1021;2001;3;0.00
199896;1035;2001;1;0.00
200139;1059;2001;4;75.00
200383;1022;2003;2;100.00
200628;1039;2003;3;100.00
200874;1090;2004;1;0.00
201121;1003;2005;4;100.00
