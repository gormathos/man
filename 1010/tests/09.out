Pascal: 45 51 14 16
Python: 41 55 15 15
C++: 34 49 18 13
Java: 36 45 13 6
C#: 35 48 10 19
